### PR TITLE
JARVIS-823 - Relaunch into OSS Update Center

### PR DESCRIPTION
**Pull request by Evildethow**
@kwhetstone @daniel-beck if you have a moment could you review and make sure I haven't missed anything?

Essentially, this PR is a `cp -Rp cloudbees/cloudbees-jenkins-advisor-plugin/* jenkinsci/cloudbees-jenkins-advisor-plugin` with some amendments to the pom.xml & README.md

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,12 +10,14 @@
...
-  <version>1.6-SNAPSHOT</version>
+  <version>2.10-SNAPSHOT</version>
   <packaging>hpi</packaging>
 
   <properties>
-    <jenkins.version>2.19.3</jenkins.version>
-    <java.level>7</java.level>
+    <jenkins.version>2.60.3</jenkins.version>
+    <java.level>8</java.level>
+    <hpi-plugin.version>2.2</hpi-plugin.version>
```
Comment (oleg-nenashev) [inline pom.xml]: YAGNI, please update the Parent POM instead

File: pom.xml [hunk 2]
```diff
@@ -10,12 +10,14 @@
...
+    <jenkins-test-harness.version>2.31</jenkins-test-harness.version>
```
Comment (oleg-nenashev) [inline pom.xml]: May be unstable in modern CI/CD flow

File: pom.xml [hunk 3]
```diff
@@ -104,37 +94,57 @@
...
       <artifactId>wiremock-standalone</artifactId>
-      <version>2.12.0</version>
+      <version>2.22.0</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>2.9.0</version>
+      <version>2.24.5</version>
```
Comment (oleg-nenashev) [inline pom.xml]: Will not be testable with Java 11, needs to take version from recent Plugin POM

File: pom.xml [hunk 4]
```diff
@@ -104,37 +94,57 @@
...
+          <groupId>org.objenesis</groupId>
+          <artifactId>objenesis</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
+
+    <dependency>
+      <groupId>org.objenesis</groupId>
+      <artifactId>objenesis</artifactId>
+      <version>3.0.1</version>
```
Comment (oleg-nenashev) [inline pom.xml]: Plugin POM dependency

File: pom.xml (hunk 3 above)
Comment (Evildethow) [reply pom.xml]: fixed

File: pom.xml (hunk 4 above)
Comment (Evildethow) [reply pom.xml]: removed

File: pom.xml (hunk 1 above)
Comment (Evildethow) [reply pom.xml]: done

File: pom.xml (hunk 2 above)
Comment (Evildethow) [reply pom.xml]: removed

**Pull request merged**